public/app/stores/gateway/gateway-store.js

```diff
@@ -9,24 +11,45 @@ module.exports = Reflux.createStore({
     this.onFetchGateway();
   },
 
+  getSetupAppUrl: function getSetupAppUrl () {
+
+    function isOSX () {
+      var userAgentString = navigator.userAgent;
+      var environment = platform.parse(userAgentString);
+      return /OS X/.test(environment.os.family);
+    }
+
+    if (isOSX(platform)) {
+      return window.yetu.config.setupDownloadUrlMac;
+    } else {
+      return window.yetu.config.setupDownloadUrlWin;
+    }
+  },
+
+  createModel: function createModel (data) {
+    return _.assign(data, {
+      setupAppUrl: this.getSetupAppUrl()
+    });
+  },
+
   getInitialState: function getInitialState () {
     // fetch gateway status on each page view
     // as we don't have polling
     this.onFetchGateway();
     return {
-      model: {}
+      model: this.createModel({})
```

Comment:
I've added a TD card to ~~cut~~ imrpove this { model, error } ~~bullshit~~ situation